src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "advancements.tconstruct.smeltery.mighty_smelting.description": "制成指南中的冶炼炉,强大的冶炼",
+  "advancements.tconstruct.smeltery.structure.title": "强大的冶炼炉",
+  "advancements.tconstruct.smeltery.structure.description": "通过铸造熔铜制作冶炼炉控制器,建造冶炼炉,然后打开冶炼炉的界面",
+  "advancements.tconstruct.smeltery.tinkers_anvil.title": "高级工具制造",
+  "advancements.tconstruct.smeltery.tinkers_anvil.description": "用冶炼炉的合金制作匠魂铁砧！",
+  "advancements.tconstruct.smeltery.tool_forge.title": "锻造工具",
+  "advancements.tconstruct.smeltery.tool_forge.description": "锻造铁砧中广泛的工具之一",
+  "advancements.tconstruct.smeltery.abilities.title": "技能大师",
+  "advancements.tconstruct.smeltery.abilities.description": "将每个默认技能修正至少应用一次在工具上",
+
+  "_comment": "Foundry Advancements",
+  "advancements.tconstruct.foundry.fantastic_foundry.title": "下界冶炼",
+  "advancements.tconstruct.foundry.fantastic_foundry.description": "梦幻铸造厂, 下界熔炼指南",
+  "advancements.tconstruct.foundry.alloyer.title": "基本冶炼",
+  "advancements.tconstruct.foundry.alloyer.description": "使用冶炼机、燃料罐、一些锭罐、铸造台、铸造盆和浇筑口创建基本的合金化",
+  "advancements.tconstruct.foundry.structure.title": "神奇的铸造厂",
+  "advancements.tconstruct.foundry.structure.description": "用合金、黑曜石制作铸造控制器,建造铸造厂,打开铸造厂 UI",
+  "advancements.tconstruct.foundry.blaze.title": "炽热之血",
+  "advancements.tconstruct.foundry.blaze.description": "用炽热的鲜血装满储罐",
+  "advancements.tconstruct.foundry.plate_armor.title": "钢甲",
+  "advancements.tconstruct.foundry.plate_armor.description": "制作全套钢甲",
+  "advancements.tconstruct.foundry.manyullyn_lanterns.title": "富的流油",
+  "advancements.tconstruct.foundry.manyullyn_lanterns.description": "制造一大堆熔炉或焦黑储罐,里面装满了熔融的玛玉灵",
+  "advancements.tconstruct.foundry.encyclopedia.title": "百科全书",
+  "advancements.tconstruct.foundry.encyclopedia.description": "用您从所有书籍中获得的知识制作材料百科全书",
+
+  "_comment": "Gadget Advancements",
+  "advancements.tconstruct.world.tinkers_gadgetry.title": "匠魂的小工具",
+  "advancements.tconstruct.world.tinkers_gadgetry.description": "制作匠魂的小工具开始探索世界",
+  "advancements.tconstruct.world.earth_island.title": "大地史莱姆岛",
+  "advancements.tconstruct.world.earth_island.description": "在海洋中找到一个大地史莱姆岛",
+  "advancements.tconstruct.world.sky_island.title": "天空史莱姆岛",
+  "advancements.tconstruct.world.sky_island.description": "在天空中找到一个天空史莱姆岛",
+  "advancements.tconstruct.world.blood_island.title": "血液史莱姆岛",
+  "advancements.tconstruct.world.blood_island.description": "在下界岩浆中中找到一个血液史莱姆岛",
+  "advancements.tconstruct.world.ender_island.title": "末影史莱姆岛",
+  "advancements.tconstruct.world.ender_island.description": "在末地找到一个末影史莱姆岛",
+  "advancements.tconstruct.world.slimesuit.title": "史莱姆套装",
+  "advancements.tconstruct.world.slimesuit.description": "制作一套完整的史莱姆装备",
+  "advancements.tconstruct.world.slimeskull.title": "头颅收藏家",
+  "advancements.tconstruct.world.slimeskull.description": "制作每一种史莱姆头",
+  "advancements.tconstruct.world.clay_island.title": "史莱姆岛",
+  "advancements.tconstruct.world.clay_island.description": "在天空中找到一个史莱姆岛",
```

此处clay应为[黏土](https://minecraft.fandom.com/zh/wiki/%E9%BB%8F%E5%9C%9F%E5%9D%97)。
```suggestion
  "advancements.tconstruct.world.clay_island.title": "黏土岛",
  "advancements.tconstruct.world.clay_island.description": "在天空中找到一个黏土岛",
```

---

这里我不是很理解，就游戏内容而言中文玩家不使用史莱姆岛来称呼它吗？据我所知先生，黏土代表完全不同物品。关于您的意见我会采纳大部分，感谢您的检查

---

> 

上文已经出现了多种类型的史莱姆岛，这个 Clay Island 应该是新版本加入的新结构，跟以往的史莱姆岛不同。
须知游戏中是已经有黏土（Clay）这个译名的，可以了解一下。如果我的猜测是正确的，你应该能在游戏里找到全新的黏土构成的空岛，形貌应该和各种史莱姆岛类似。